spring-cloud-kubernetes-test-support/src/main/java/org/springframework/cloud/kubernetes/integration/tests/commons/Fabric8Utils.java

```diff
@@ -90,7 +92,37 @@ public static void setUp(KubernetesClient client, String namespace) throws Excep
 		innerSetup(client, namespace, serviceAccountAsStream, roleBindingAsStream, roleAsStream);
 	}
 
-	public static void setUpIstio(KubernetesClient client, String namespace) throws Exception {
+	public static void setUpClusterWide(KubernetesClient client, String serviceAccountNamespace,
+			Set<String> namespaces) {
+		InputStream clusterRoleBindingAsStream = inputStream("cluster/cluster-role.yaml");
```

Comment:
a cluster role set-up, so that this is simpler, but also more usable for other future cases